lib/msf/core/post/file.rb

```diff
@@ -126,6 +126,8 @@ def directory?(path)
       stat = session.fs.file.stat(path) rescue nil
       return false unless stat
       return stat.directory?
+    elsif session.type == 'powershell'
+      return cmd_exec("(Get-Item #{path}).PSIsContainer")&.include?("True")
```

Comment:
This looks like it'll break if `path` contains spaces since it's not quoted like the shell one.

Reply:
okay, I'll add quotes :+1: